jetson/percep/pcl.cpp

```diff
@@ -186,177 +186,81 @@ void PCL::FindInterestPoints(std::vector<pcl::PointIndices> &cluster_indices,
 
 }
 
+/* --- Get Angle --- */
+//This function finds the angle off the x axis the
+//line that passes through both these points is
+//Direction of 0 is left and 1 is right
+double PCL::getAngle(int buffer, int direction, std::vector<std::vector<int>> &interest_points,
+                    shared_ptr<pcl::visualization::PCLVisualizer> viewer, std::vector<int> &obstacles){
+    double newAngle = 0;
+    //If Center Path is blocked check left until have to turn too far
+    while(newAngle > -70 && newAngle < 70){
+        
+        //Finding angle off center
+        double xDist = pt_cloud_ptr->points[obstacles.at(direction)].x;
+        double zDist = pt_cloud_ptr->points[obstacles.at(0)].z;//Length of adjacent
+        xDist += direction ? buffer+HALF_ROVER : -(buffer+HALF_ROVER); //Calculate length of opposite
+        newAngle = atan(xDist/zDist)*180/PI;//tan-1(opposite/adjacent)
+
+        //Create compareLine Functors
+        compareLine leftLine (newAngle, -HALF_ROVER);
+        compareLine rightLine (newAngle, HALF_ROVER);
+        
+        obstacles.clear();
+        
+        if(CheckPath(interest_points, viewer, obstacles, leftLine, rightLine)){
+            #if PERCEPTION_DEBUG
+            std::cout<<"!!!!!!!!!!!!!!!!!!!!!!!!!!!!!FOUND NEW PATH AT: "<<newAngle<<std::endl;
```

Comment:
Are all these exclamation marks really necessary? Maybe a long term goal should be to improve our debugging mode for perception. Also, what are your thoughts on just using verbose flag so we don't need to rebuild the service?

Reply:
https://github.com/umrover/mrover-workspace/issues/460

Reply:
You mean use a command line arg as opposed to a build arg? Definitely a move

Reply:
I'll make an issue for the second part of your question